src/models/tools/text/text-content.ts

```diff
@@ -64,6 +65,18 @@ export const TextContentModel = types
       return Value.fromJSON(parsed);
     }
   }))
+  .views(self => ({
+    convertSlate() {
```

Comment:
What's the reason for not calling this `getText`? Since it potentially handles other formats, too

Reply:
It converts whatever format it's currently in to slate format -- it could be named `convertToSlate()`. I wanted to be explicit about the format being returned in case we ever decide to support other document formats.